.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Fixes #
```

Comment:
Most of the time there's no associated issue since we're writing the issues (a.k.a Tasks) in the GitLab reflectometry repo. Thus, I fear this will end up becoming more of a nuisance than a facilitator. You could insert it within a comment block, though